src/pages/index.js

```diff
@@ -80,6 +56,165 @@ const cardLinkInput = addCardFormElement.querySelector(
 );
 const cardSelector = "#card-template";
 
+// Likes Api
+
+function handleCardLikeClick(cardId, isLiked) {
+  if (isLiked) {
+    api
+      .unlikeCard(cardId)
+      .then((cardData) => {
+        this._likes = cardData.isLiked;
+        this._renderLikes();
```

Comment:
`_renderLikes` and other methods


Please, delete the **low dash** from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.


**Please, correct it everywhere**